lua/neo-tree/sources/common/components.lua

```diff
@@ -275,7 +275,11 @@ M.icon = function(config, node, state)
     elseif node:is_expanded() then
       icon = config.folder_open or "-"
     else
-      icon = config.folder_closed or "+"
+      if node.is_link then
+        icon = config.folder_symlink
```

Comment:
```suggestion
        icon = config.folder_symlink or config.folder_closed or "+"
```

This needs fallbacks.